cluster/state/cluster.go

```diff
@@ -4,5 +4,24 @@ package state
 
 // Cluster represents the state of a cluster after applying a sequence of mutations.
 type Cluster struct {
-	// TODO(corver): Implement
+	Name         string
+	Threshold    int
+	DKGAlgorithm string
+	ForkVersion  []byte
```

Comment:
nit: why not use `Network string`?

Reply:
I'm using the `cluster.Definition` types